spring-data-jdbc/src/test/java/org/springframework/data/jdbc/repository/JdbcRepositoryIdGenerationIntegrationTests.java

```diff
@@ -107,15 +131,126 @@ void manuallyGeneratedIdForSaveAll() {
 		assertThat(immutableWithManualIdEntityRepository.findAll()).hasSize(2);
 	}
 
+	@Test // DATAJDBC-2003
+    @Sql(statements = "INSERT INTO EntityWithSeq(id, name) VALUES(1, 'initial value');")
+	void testUpdateAggregateWithSequence() {
+
+        // given.
+        EntityWithSeq entityWithSeq = new EntityWithSeq();
+        entityWithSeq.id = 1L;
+        entityWithSeq.name = "New name";
+
+        // when.
+        EntityWithSeq updated = entityWithSeqRepository.save(entityWithSeq);
+
+        // then.
+        auditableNamedParameterJdbcTemplate.assertSequenceOfStatements(UPDATE);
+        Assertions.assertThat(updated.id).isEqualTo(1L);
```

Comment:
We use static imports for `assertThat(…)`

Reply:
Done, agreed